rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -26,6 +29,10 @@
 
 logger = logging.getLogger(__name__)
 
+TF_HUB_MODULE_URL = (
+    "https://github.com/PolyAI-LDN/polyai-models/releases/download/v1.0/model.tar.gz"
+)
```

Comment:
I think we should not point to that URL anymore as it does not exist any longer. I think @dakshvar22 was working on a solution.

Reply:
@dakshvar22 let me know what you'd like me to do

Reply:
Yeah that's a separate PR which I'll submit today, let's leave it as it is here.